pkg/cluster/cluster_config.go

```
@@ -257,5 +265,48 @@ func getClusterInfo(ctx context.Context, cli client.Client) (ClusterInfo, error)
 	}
 	c.Version = ocpVersion
 
+	// Check for FIPs
+	if fipsEnabled, err := isFipsEnabled(ctx, cli); err == nil {
+		c.FipsEnabled = fipsEnabled
+	} else {
+		logf.FromContext(ctx).Info("could not determine FIPS status, defaulting to false", "error", err)
+	}
+
 	return c, nil
 }
+
+func isFipsEnabled(ctx context.Context, cli client.Client) (bool, error) {
+    // Check the install-config for the fips flag and it's value
+    // https://access.redhat.com/solutions/6525331
+	cm := &corev1.ConfigMap{}
+	namespacedName := types.NamespacedName{
+		Name:      "cluster-config-v1",
+		Namespace: "kube-system",
+	}
+
+	if err := cli.Get(ctx, namespacedName, cm); err != nil {
+		return false, fmt.Errorf("failed to get ConfigMap %s/%s: %w", namespacedName.Namespace, namespacedName.Name, err)
+	}
+
+	if installConfigStr, ok := cm.Data["install-config"]; ok {
```

it is generally better to fail fast, something like:

```go
installConfigStr := cm.Data["install-config"]
if installConfigStr == "" {
     // No install-config found or empty
     return false, nil
}
```